src/Tribe/Aggregator/Processes/Batch_Imports.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+class Tribe__Events__Aggregator__Processes__Batch_Imports {
+	public function hook() {
+		add_filter( 'tribe_events_aggregator_build_url', [ $this, 'build_url' ], 10, 3 );
+		add_filter( 'tribe_aggregator_service_post_import_args', [ $this, 'import_args' ], 10, 2 );
+		add_filter( 'tribe_aggregator_allow_batch_push', [ $this, 'allow_batch_import' ], 10, 2 );
+	}
+
+	public function build_url( $url, $endpoint, $api ) {
+		if ( $endpoint !== 'import' ) {
+			return $url;
+		}
+
+		return "{$api->domain}{$api->path}{v2.0.0}/{$endpoint}";
+	}
+
+	public function allow_batch_import( $service_supports_batch_push, $abstract ) {
+		if ( ! $service_supports_batch_push ) {
```

Comment:
:no_entry_sign: Missing doc comment for function allow_batch_import()

<sub>hash: f03c7d1cd902ef9004e3c65b33172cd8</sub>